### DISCUSSION TITLE
Why does modal example .css introduces new class, when built-in could be used?

### DISCUSSION THREAD

**Discussion by schamberg97** [twbs/bootstrap#34954] [General]
Hey there! I have a very minor question. I have been looking at bootstrap examples for inspiration and have noticed that modals example uses a new class `border-right` in the third example modal. I am not sure, whether there is any specific need for it to exist, because a very similar effect could be introduced with built-in `border-end` class. Am I missing something or should I make a PR?

**Comment by ghost** [comment]
Please provide the link as of where it is used.
As far as I know, making new classes can provide us with more options to customize.
I see border-end{border-right:1px solid;} on css page
